docs/faq.rst

```diff
@@ -129,3 +129,54 @@ setting the ``expect`` request option to ``false``:
 
     // Disable the expect header on all client requests
     $client = new GuzzleHttp\Client(['expect' => false]);
+
+How can I track a redirected requests HTTP codes?
+============================================
+
+You can enable the `track_redirects` option which will enable Guzzle to track
+any redirected URI and status code. Each redirected URI will be stored in the
+``X-Guzzle-Redirect-History`` header and each Response Code will be stored in
+the ``X-Guzzle-Redirect-Status-History`` header.
+
+The ``X-Guzzle-Redirect-History`` header will exclude the initial request's URI
```

Comment:
Instead of the header names we could say "the URL history" and "the status code history". Just to avoid duplication. But I am open to other suggestions.